src/NServiceBus.AcceptanceTests/Serialization/When_skip_wrapping_xml.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace NServiceBus.AcceptanceTests.Serialization
+{
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Xml.Linq;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using MessageMutator;
+    using NUnit.Framework;
+
+    public class When_skip_wrapping_xml : NServiceBusAcceptanceTest
```

Comment:
so a test testing that the xml containing invalid characters fails deserialization and moves to the error queue? not sure, is that worth it since it's a violation of the xml spec?